content/blog/configure-isut-and-ams-for-vlcm-based-firmware-updates-on-gen-12-servers.md

```diff
@@ -0,0 +1,98 @@
+---
+title: How to Configure iSUT and AMS for vLCM-Based Firmware Updates on HPE
```

Comment:
Branding required sentence case for the title.
Therefore, I would suggest the following for the title:

How to configure iSUT and AMS for vLCM-Based firmware updates on HPE Gen12 servers (High Security Mode)